sqlglot/__init__.py

```diff
@@ -35,8 +36,10 @@
 __version__ = "10.5.6"
 
 pretty = False
+"""Whether to format generated SQL by default."""
```

Comment:
is this a pdocs thing? Docstrings under a variable?

Reply:
Yep, it makes the variable appear in the API documentation:

<img width="738" alt="Screenshot 2023-01-26 at 6 53 16 PM" src="https://user-images.githubusercontent.com/46752250/214898282-28b9e5e9-016e-4ef0-a39b-9af79ad0f6c7.png">